internal/aez/verification_test.go

```diff
@@ -0,0 +1,350 @@
+package aez
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestTCAcceptSuccess_CSVSpecifications(t *testing.T) {
```

Comment:
I've been thinking and perhaps we are a bit too eager to use these automated test structures when the entire body probably can be reduced to:

```go
tcv := TCAcceptSuccess{}
want := []string{"Specification", Specification}
if got := tcv.CSVSpecification(); !reflect.DeepEqual(got, want) {
  t.Errorf("TCAcceptSuccess.CSVSpecifications() = %v, want %v", got, want)
}
```

I think we only should use this automatic content as is when we have parameterized tests or other reasons it makes sense. They are still a good start in writing above code.

This applies to several tests.